### PR TITLE
Anim last frame and a bunch more

### PR DESCRIPTION
**Pull request by JoannesJ**
I applied a meh fix for the focus stealing issue MemeLabs/Rustla2/issues/117
Far from perfect, but I consider it better than nothing.
This fix does guarantee that focus will be stolen from dgg chat instead.
@xDashh Just can it if you don't like that idea.

### CODE REVIEW

**Comment by xDashh** [comment]
I'm not sure if I can follow the meaning/consequences of that bug completely. If you think the fix is beneficial for us, I'd just keep it.

Might merge later today if I do more emote work, otherwise after review.

**Comment by JoannesJ** [comment]
It doesn't really fix the bug, it rather shifts the behaviour to be more favorable towards sgg chat in winning the focus battle.
Whether it's really beneficial, eh... 

I think the thing to consider is that someone with dgg chat open will always have sgg chat loaded, where that isn't the case in reverse.
So in that regard, complaints might increase.

Yeah, I just changed my opinion to canning it Hhehhehe

File: assets/anim.scss [hunk 1]
```diff
@@ -294,20 +294,33 @@ $spritesDir : '../../emotes/emoticons-animated/';
...
         $repeat,
+        $end-on-last-frame: false,
         $file-ext: '.png') {
 
     $path: $spritesDir + $file-name + $file-ext;
+    $frame-width: $sprite-width / $frame-count;
+
+    @if $frame-width != round($frame-width) {
+        @error
+            "$sprite-width should be dividable by $frame-count to prevent blurry rendering
```
Comment (KenanY) [inline assets/anim.scss]: nitpick:
```suggestion
            "$sprite-width should be divisible by $frame-count to prevent blurry rendering
```

**Comment by KenanY** [comment]
(squash before merging please 🙏 )

**Comment by JoannesJ** [comment]
What is the philosophy on the scope of commits here?

Ready to go, btw 👍

**Comment by KenanY** [comment]
Erm, looking at it now I realize the commits are actually well-scoped as-is. Scratch that squash 👍

**Pull request merged**